web/models.py

```
@@ -660,7 +660,7 @@ class EducationalVideo(models.Model):
     """Model for educational videos shared by users."""
 
     title = models.CharField(max_length=200)
-    description = models.TextField()
+    description = models.TextField(blank=True, help_text="Optional – describe what viewers will learn from this video")
```

Ruff (0.8.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)